### PR TITLE
Update PPP and UG

### PR DESCRIPTION
**Pull request by weilin1202**
This PR updates:
- my individual PPP
- adding 'event' in front of all occurrences of 'tag' 
- Specify that `evt\` and `evt\EVENT_INDEX` cannot be in the same command

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -109,14 +109,14 @@ Format: `list`
...
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [evt/EVENT INDEX]…​`
+Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [evt/EVENT_INDEX]…​`
 
 * Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
 * At least one of the optional fields must be provided.
 * Existing values will be updated to the input values.
-* When editing tags, the tags will be added to the existing tags of the person i.e adding of tags is cumulative.
-* You can remove all the person’s event tags by typing `evt/` without
-    specifying any event index after it.
+* When editing event tags, the event tags will be added to the existing event tags of the person i.e adding of event tags is cumulative.
```
Comment (venuslimm) [inline docs/UserGuide.md:117]:
```suggestion
* When editing event tags, the event tags will be added to the existing event tags of the person i.e. adding of event tags is cumulative.
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -109,14 +109,14 @@ Format: `list`
...
+* You can remove all the person’s event tags by typing `evt/` only without specifying any event index after it. 
+* Either the addition of event tags or the removal of event tags can be done at a time. ie removal and addition of event tags cannot be done in a single command. eg. `evt/ evt/2` and `evt/2 evt/` cannot be recognised.
```
Comment (venuslimm) [inline docs/UserGuide.md:119]:
```suggestion
* Either the addition of event tags or the removal of event tags can be done at a time, i.e. removal and addition of event tags cannot be done in a single command. For example, `evt/ evt/2` cannot be recognised as removing all the event tags and adding event index 2 and `evt/2 evt/` cannot be recognised as adding event index 2 then removing all the event tags. 
```

File: docs/team/weilin1202.md [hunk 3]
```diff
@@ -10,36 +10,39 @@ It provides a centralised platform for them to manage their work, contacts, and
...
-  * Justification: to be added soon
-  * Highlights: to be added soon
-  * Credits: *{to be added soon}*
-
-* **New Feature**: to be added soon
-  * What it does: to be added soon
-  * Justification: to be added soon
-  * Highlights: to be added soon
-  * Credits: *{to be added soon}*
+* **New Feature**: Added an event tagging feature to the `add` and `edit` command
```
Comment (venuslimm) [inline docs/team/weilin1202.md:13]: I think you can include the link to the PR here!

File: docs/team/weilin1202.md [hunk 4]
```diff
@@ -10,36 +10,39 @@ It provides a centralised platform for them to manage their work, contacts, and
...
-  * Justification: to be added soon
-  * Highlights: to be added soon
-  * Credits: *{to be added soon}*
+* **New Feature**: Added an event tagging feature to the `add` and `edit` command 
+  * What it does: Allows user to tag an existing event to a person 
+  * Justification: This improves the product because a user can associate events to persons in the event planner. 
+  * Highlights: While this is similar to the previous `tag` feature, user can add an event tagged to a person through the event index, user need not enter the entire event name.
+
+
+* **New Feature**: Added a `listevcontact` command to list all contacts associated to an event
```
Comment (venuslimm) [inline docs/team/weilin1202.md:19]: This one as well 😄

**Review by venuslimm — COMMENTED** [review]
I left some suggestions 😃

File: docs/UserGuide.md (hunk 1 above)
Comment (weilin1202) [reply docs/UserGuide.md]: Edited!

File: docs/UserGuide.md (hunk 2 above)
Comment (weilin1202) [reply docs/UserGuide.md]: Edited!

File: docs/team/weilin1202.md (hunk 3 above)
Comment (weilin1202) [reply docs/team/weilin1202.md]: Added!

File: docs/team/weilin1202.md (hunk 4 above)
Comment (weilin1202) [reply docs/team/weilin1202.md]: Added :)

**Review by venuslimm — APPROVED** [review]
LGTM!

**Review by Irminrics — APPROVED** [review]
LGTM!

**Pull request merged**